app/src/main/java/com/android/voyageur/ui/trip/schedule/ScheduleScreen.kt

```diff
@@ -30,42 +29,62 @@ fun ScheduleScreen(
-              style = MaterialTheme.typography.bodyMedium,
-              color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f))
-          TextButton(
-              onClick = { navigationActions.getNavigationState().isDailyViewSelected = false }) {
+          Row(
+              modifier = Modifier.fillMaxWidth().padding(bottom = 8.dp, end = 16.dp),
+              horizontalArrangement = Arrangement.End,
+              verticalAlignment = Alignment.CenterVertically) {
+                TextButton(
+                    onClick = {
+                      navigationActions.getNavigationState().isDailyViewSelected = true
+                    }) {
+                      Text(
+                          text = "Daily",
+                          style = MaterialTheme.typography.bodyMedium,
+                          color =
+                              if (navigationActions.getNavigationState().isDailyViewSelected)
+                                  MaterialTheme.colorScheme.primary
+                              else MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f),
+                          fontWeight =
+                              if (navigationActions.getNavigationState().isDailyViewSelected)
+                                  FontWeight.Bold
+                              else FontWeight.Normal)
+                    }
                 Text(
-                    text = "Weekly",
+                    text = " / ",
                     style = MaterialTheme.typography.bodyMedium,
-                    color =
-                        if (!navigationActions.getNavigationState().isDailyViewSelected)
-                            MaterialTheme.colorScheme.primary
-                        else MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f),
-                    fontWeight =
-                        if (!navigationActions.getNavigationState().isDailyViewSelected)
-                            FontWeight.Bold
-                        else FontWeight.Normal)
+                    color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f))
+                TextButton(
+                    onClick = {
+                      navigationActions.getNavigationState().isDailyViewSelected = false
+                    }) {
+                      Text(
+                          text = "Weekly",
+                          style = MaterialTheme.typography.bodyMedium,
+                          color =
+                              if (!navigationActions.getNavigationState().isDailyViewSelected)
+                                  MaterialTheme.colorScheme.primary
+                              else MaterialTheme.colorScheme.onSurface.copy(alpha = 0.6f),
+                          fontWeight =
+                              if (!navigationActions.getNavigationState().isDailyViewSelected)
+                                  FontWeight.Bold
+                              else FontWeight.Normal)
+                    }
```

Comment:
The code for the 'Daily' and 'Weekly' buttons is nearly identical, consider extracting the shared logic into a reusable composable. This will make the code cleaner and easier to update in the future.

Reply:
Also, consider adding contentDescription or using Modifier.semantics for the text buttons to enhance accessibility. This will make the UI more user-friendly for screen readers and improve the experience for users with different accessibility needs.

Reply:
This PR didn't touch that part of the code (except for putting the Daily and Weekly buttons into a row with another button). I think this should be done in a different PR

Reply:
I am sorry for the confusion—it showed up in the modified files section, which led to my misunderstanding. I agree that this should be addressed in a separate PR.